### DISCUSSION TITLE
How to truncate text inside tooltip without using `overflow:hidden`?

### DISCUSSION THREAD

**Discussion by aaayumi** [floating-ui/floating-ui#2340] [Q&A] [answered]
I'm trying to truncate text in tooltip component but `overflow:hidden` will hide arrow svg.
Is it possible to truncate text inside tooltip while keeping arrow svg?

css
```
.tooltip {
  max-width: 200px;
  max-height: 70px;
  overflow: hidden;
  word-wrap: break-word;
  text-overflow: ellipsis;
}
```

**Comment by atomiks** [comment, marked as answer]
Add an extra div inside, with the arrow outside, and apply the truncation CSS to that
